src/main/kotlin/uk/gov/justice/digital/hmpps/authorizationserver/service/ClientsService.kt

```diff
@@ -81,6 +81,9 @@ class ClientsService(
     )
   }
 
+  private fun getMaxLastAccessedDate(clientList: List<Client>) = clientList.map { it.latestClientCredentialsAuthorization }
```

Comment:
Suggest renaming this function to `getMostRecentAccessedDate`. Also rename the parameter to `duplicateClients`, may make it clearer?